### PR TITLE
Add undo support for appointment commands

### LINKED ISSUE #70: As a user, I want to be able to undo my appointment commands

### PR DESCRIPTION
**Pull request by rickkoh**
Resolves #70

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddNoteCommand.java [hunk 1]
```diff
@@ -18,32 +18,33 @@
...
+            + PREFIX_DATE + "DATE<DD-MM-YYYY> "
+            + PREFIX_TIME + "TIME<HHMM> "
+            + PREFIX_NOTE + "NOTE "
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_DATE + "19-02-2024 "
+            + PREFIX_TIME + "1430 "
+            + PREFIX_NOTE + "General Flu ";
 
     public static final String MESSAGE_SUCCESS = "New appointment note added: %1$s";
+    public static final String MESSAGE_UNDO_ADD_SUCCESS = "Add patient medical record undone.";
```
Comment (sebin0817) [inline src/main/java/seedu/address/logic/commands/AddNoteCommand.java:39]: Think it should be `Add appointment note undone.`

**Review by sebin0817 — APPROVED** [review]
Other than modifying `MESSAGE_UNDO_ADD_SUCCESS`, LGTM

**Review by jmsandiegoo — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddNoteCommand.java (hunk 1 above)
Comment (rickkoh) [reply src/main/java/seedu/address/logic/commands/AddNoteCommand.java:39]: Hi Sebin, thanks for catching the typo!

**Pull request merged**